pkg/cvo/status.go

```diff
@@ -351,18 +356,33 @@ func (optr *Operator) syncStatus(ctx context.Context, original, config *configv1
 // unavailable AND the general payload task making progress towards its goal. An operator is given 40 minutes since
 // its last update to go ready, or an hour has elapsed since the update began, before the condition is ignored.
 func convertErrorToProgressing(history []configv1.UpdateHistory, now time.Time, status *SyncWorkerStatus) (reason string, message string, ok bool) {
-	if len(history) == 0 || status.Failure == nil || status.Reconciling || status.LastProgress.IsZero() {
-		return "", "", false
-	}
-	if now.Sub(status.LastProgress) > 40*time.Minute || now.Sub(history[0].StartedTime.Time) > time.Hour {
+	if len(history) == 0 || status.Failure == nil || status.Reconciling {
 		return "", "", false
 	}
 	uErr, ok := status.Failure.(*payload.UpdateError)
 	if !ok {
 		return "", "", false
 	}
 	if uErr.Reason == "ClusterOperatorNotAvailable" || uErr.Reason == "ClusterOperatorsNotAvailable" {
```

Comment:
We might need to include `ClusterOperator(s)Degraded` in here too?

Reply:
reworked per our discussion